### PR TITLE
Added another UI errata test and fixed compatibility with Python 3

### LINKED ISSUE #3726: Make `tests/foreman/ui/test_errata.py` compatible with both Python 2 and 3
The `nested` from `contextlib` is not available anymore on Python 3. That is causing an error when running the tests using Python 3.

```console
$ python --version
Python 3.4.3
$ python -c 'from contextlib import nested'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: cannot import name 'nested'
```

### PR DESCRIPTION
**Pull request by abalakh**
* The `nested` from `contextlib` is not available anymore on Python 3. And, basically, as we do not support python2.6 we don't need `nested`at all so i've removed it.
Closes #3726
* Implemented another stub, although it appears to be blocked by a BZ.

### CODE REVIEW

**Comment by abalakh** [comment]
No test results as the new test is skipped and another affected test is failing on jenkins for a different reason.

File: tests/foreman/ui/test_errata.py [hunk 1]
```diff
@@ -260,10 +260,8 @@ def test_positive_apply_for_all_hosts(self):
 
         @CaseLevel: System
         """
-        with nested(
-                VirtualMachine(distro=DISTRO_RHEL7),
-                VirtualMachine(distro=DISTRO_RHEL7)
-        ) as (client1, client2):
+        with VirtualMachine(distro=DISTRO_RHEL7) as client1, VirtualMachine(
+                distro=DISTRO_RHEL7) as client2:
```
Comment (sghai) [inline tests/foreman/ui/test_errata.py]: :+1:

**Comment by sghai** [comment]
ACK

**Comment by svtkachenko** [comment]
ACK with small nitpick.

**Pull request merged**